cli.js

```diff
@@ -507,6 +510,11 @@ process.on('uncaughtException', function(err) {
       globalConfig.set(property, value);
       break;
 
+    case 'cache':
```

Comment:
In line with the above, perhaps lets make it `jspm cache-clear`, and `jspm cc` for short?